app/src/main/java/com/example/plantsapp/presentation/ui/plantdetail/PlantDetailFragment.kt

```diff
@@ -13,7 +13,7 @@ import com.example.plantsapp.domain.model.Plant
 class PlantDetailFragment : Fragment(R.layout.fragment_plant_detail) {
 
     private val binding: FragmentPlantDetailBinding by viewBinding(FragmentPlantDetailBinding::bind)
-    private val detailViewModel: PlantDetailViewModel by viewModels {
+    private val detailViewModel: PlantDetailViewModel by activityViewModels {
```

Comment:
It is not the best approach - because right now your viewmodel is bound to activity, not to fragment (and alive even when our fragment is not needed)